data_validation/consts.py

```diff
@@ -90,6 +90,8 @@
     CUSTOM_QUERY,
 ]
 
+# Partition types
+PARTITION_TYPES = ["primary_key", "primary_key_mod", "hash_mod"]
```

Comment:
We can remove this and line 71 since it's unused for now.

Reply:
Done